crates/nu-command/src/filesystem/ls.rs

```diff
@@ -219,6 +225,69 @@ impl Command for Ls {
     }
 }
 
+impl Ls {
+    fn apply_post_sorting(
+        engine_state: &EngineState,
+        stack: &mut Stack,
+        call: &Call,
+        result: PipelineData,
+    ) -> Result<PipelineData, ShellError> {
+        let sort_by_configs = stack.get_config(engine_state).ls.sort_by.clone();
```

Comment:
I think you should be able to avoid this clone, since you only pass it by reference later